src/main/java/com/dev/cinema/dao/impl/CinemaHallDaoImpl.java

```diff
@@ -52,4 +55,19 @@ public CinemaHall add(CinemaHall cinemaHall) {
             throw new DataProcessingException("Can't get all cinema hall", e);
         }
     }
+
+    @Override
+    public CinemaHall getByCinemaHallId(Long id) {
```

Comment:
```suggestion
    public CinemaHall getById(Long id) {
```
you are making these methods in corresponding daos, so there is no reason to specify which id in method's name(same everywhere)